data/moves.ts

```diff
@@ -13716,11 +13713,9 @@ export const BattleMovedex: {[moveid: string]: MoveData} = {
 		pp: 5,
 		priority: 0,
 		flags: {protect: 1, mirror: 1},
-		onPrepareHit(target, source, move) {
+		onTryHit(target, source, move) {
 			if (source.ignoringItem()) return false;
-			const item = source.getItem();
-			if (!this.singleEvent('TakeItem', item, source.itemData, source, source, move, item)) return false;
-			if (!item.id) return false;
+			if (!target.item) return false;
```

Comment:
The description is no longer consistent with this change. (I guess you just need the shortDesc now?)